create3_control/controller_server.py

```diff
@@ -16,15 +28,16 @@
 
 import create3_control.controllers.factory as controllers_factory
 
+
 class ControllerServer(Node):
 
     def __init__(self):
         super().__init__('controller')
 
         # Setup ROS 2 parameters
         self.declare_parameter('pose_topic', 'odom')
-        self.declare_parameter('controller_type', 'polar_coordinates')
-        self.declare_parameter('control_period', 0.015)
+        self.declare_parameter('ctrl_type', 'polar_coordinates')
```

Comment:
avoid abbreviations unless necessary.
this is especially true in external-facing values (such as ROS parameters)

Reply:
ok